### DISCUSSION TITLE
read-only question

### DISCUSSION THREAD

**Discussion by schy3k** [FreeRDP/FreeRDP#7968] [Q&A]
HI.

How do I set read-only on the drive channel?

/drive:NAME,PATH

**Comment by akallabeth** [comment]
not supported by freerdp.

There are ways to achieve that though:
* you can always use read only permissions locally (e.g. redirect folder `foo` which has permissions `dr-xr-xr-x` and (in best case) is not owned by the user you have the rdp client running with
* you can redirect a folder on a read only mount
